packages/playwright-core/src/server/browserContext.ts

```diff
@@ -276,6 +276,22 @@ export abstract class BrowserContext extends SdkObject {
     return await this.doGetCookies(urls as string[]);
   }
 
+  async removeCookies(cookies: {name?: string, domain?: string, path?: string}): Promise<void> {
+    const setCookies = await this.cookies();
+
+    if (!cookies.name && !cookies.domain && !cookies.path)
```

Comment:
the call above is a waste if you will early return here
maybe moving the `await  this.cookies()` under?

Reply:
This should probably throw 'Either name, domain or path are required'. Same should be mentioned in the doc (md file).